From 5730b9b6eb638859a13613becbd1d3d82bcf7a92 Mon Sep 17 00:00:00 2001
From: liyong4x <yongx.a.li@intel.com>
Date: Tue, 12 Jun 2018 04:34:38 -0400
Subject: [PATCH 092/129] media: intel-ipu4: max9286: Correct pclk

Message for Open Source:
The previous pclk value can work normally, but not recommended.
So change as the right values from sensor vendor.

Message for Internal:
The previous pclk value can work normally, but not recommended.
So change as the right values from sensor vendor.

[Issue/Feature] The previous pclk value are not recommended
by vendor.

[Root Cause/Changes] change the max9286_op_sys_clock as 87750000.

Change-Id: I66695ec5fd46d0238184d3636d85f8724531f750
Tracked-On: #JIIAP-475
Signed-off-by: liyong4x <yongx.a.li@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/max9286.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/max9286.c b/drivers/media/i2c/max9286.c
index 71aff37..6a472f3 100644
--- a/drivers/media/i2c/max9286.c
+++ b/drivers/media/i2c/max9286.c
@@ -655,7 +655,7 @@ static int max9286_s_ctrl(struct v4l2_ctrl *ctrl)
 	.s_ctrl = max9286_s_ctrl,
 };
 
-static const s64 max9286_op_sys_clock[] = { 264000000, };
+static const s64 max9286_op_sys_clock[] = { 87750000, };
 static const struct v4l2_ctrl_config max9286_controls[] = {
 	{
 		.ops = &max9286_ctrl_ops,
-- 
1.9.1

